RestaurantReservation.Db/RepositoriesInterfaces/IEmployeeRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using RestaurantReservation.Db.DataModels;
+
+namespace RestaurantReservation.Db.RepositoriesInterfaces
+{
+    public interface IEmployeeRepository
+    {
+        Task CreateEmployeeAsync(Employee employee);
+        Task DeleteEmployeeAsync(int employeeId);
+        Task<List<EmployeeView>> EmployeeViewAsync();
```

Comment:
Method name should be 'Verb' 
suggestions ListEmployeeViewAsync();
or GettEmployeeViewAsync()